pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -1307,7 +1308,8 @@ public void testCaseInsensitivity() {
 
     for (String query : queries) {
       try {
-        postQuery(query);
+        JsonNode response = postSqlQuery(query);
```

Comment:
Can you please check why it does not apply to pql? It should apply to both pql and sql

Reply:
It seems the table name is set from the resource name, which made the table cache failed to get the correct schema to compare. I've included the fix in this PR.